File: exercicios em C/q20.cpp
#include <stdio.h>

int main(void){
	int n, i, num, menor, maior;
	num = 0;
	printf("\n\nEste programa ler uma quantidade qualquer de numero e determina o maior e menor numero.");
	printf("\n\nDigite a quantidade de valores a qual voce deseja comparalos: ");
	scanf("%d", &n);
	for (i = 1; i <= n; i++){
		printf("\n\nDigite o %d valor: ", i);
		scanf("%d", &num);
		if (i == 1){
			maior = num;
			menor = num;
		}
		else
			if (num > maior){
				maior = num;
			}
			if(num < menor){
				menor = num;
			}
	}
	printf("\n\nSeu maior numero e %d, e seu  menor e %d.", maior, menor);
	printf("\n\nObrgado por utilizar nosso programa.");
	return 0;
}

File: exercicios em Python/q20.py
import os
print("\nEste programa ler uma quantidade qualquer de numeros e determina qual e maior e qual e menor!")
n = int(input("\nDigite o numero de valores que ira usar: "))
i = 1
while i <= n:
        print("\nDigite o ",i," valor")
        num = int(input(""))
        if i == 1:
                num1 = num
                num2 = num
        elif num1 < num:
                num1 = num
        elif num2 > num:
                num2 = num
        i = i + 1

print("\nO maior valor e ",num1,",  e o menor e ",num2)
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")
